### PR TITLE
Show "enabling..." on featured experiment enable

### LINKED ISSUE #3267: [Featured Experiment] The "Enable Notes" green button animation when clicked, wrongly states "Installing..." instead of "Enabling..."
**[Affected versions]:**
- Firefox v57.0.4 and above

**[Affected Platforms]:**
- All Windows
- All Mac OS
- All Linux

**[Prerequisites]:**
- Following 5 prefs have been set or added on a new profile: "extensions.legacy.enabled = true" "xpinstall.signatures.required = false" "testpilot.env = dev" "extensions.webapi.testing = true" "extensions.install.requireBuiltInCerts = false".
- test Pilot add-on was already installed on the same profile.

**[Steps to reproduce]:**
1. Open Firefox browser with the profile from prerequisites and navigate to https://testpilot.dev.mozaws.net/experiments page.
2. Click the "Enable Notes" green button while observing its animation.

**[Expected result]:**
- A barberpole animation appears on the button with "Enabling..." string 

**[Actual result]:**
- A barberpole animation appears on the button but wrongly displays "Installing..." string.

**[Notes]**
- This issue occurs also on Stage server.
- Here is a screen recording of the issue: https://goo.gl/qUaT5o.

### PR DESCRIPTION
**Pull request by meandavejustice**
- fixes #3267

### CODE REVIEW

File: frontend/src/app/components/MainInstallButton/index.js [hunk 1]
```diff
@@ -123,13 +129,11 @@ export default class MainInstallButton extends React.Component {
...
     const makeInstallButton = (extraClass = "") => {
+      const isEnabling = (experimentTitle && hasAddon && !isExperimentEnabled(experiment) && isInstalling);
       return <button onClick={e => this.install(e)}
         className={classnames(`button primary main-install__button ${extraClass}`, { "state-change": isInstalling })}>
         {!isInstalling && installButton}
-        {isInstalling &&
-          <Localized id="landingInstallingButton">
-            <span className="progress-btn-msg">Installing...</span>
-          </Localized>}
+        {isInstalling ? isEnabling ? enablingButton : installingButton : null} {/* eslint-disable-line no-nested-ternary */}
```
Comment (meandavejustice) [inline frontend/src/app/components/MainInstallButton/index.js]: This seems like a reasonable case to make an exception for the nested ternary rule.

File: locales/en-US/app.ftl [hunk 2]
```diff
@@ -37,6 +37,7 @@ landingLegalNotice = By proceeding, you agree to the <a>Terms of Use</a> and <a>
 [[landingInstall]]
 landingInstallButton = Install the Test Pilot Add-on
 landingInstallingButton = Installing...
+landingEnablingButton = Enabling...
```
Comment (flodolo) [inline locales/en-US/app.ftl]: Please use a single unicode `…` instead of `...`. Fixing the old string would be great

Comment (meandavejustice) [reply locales/en-US/app.ftl]: @flodolo will I need to update the key for `landingInstallingButton` for this change?

Comment (flodolo) [reply locales/en-US/app.ftl]: No need for new ID, each locale should have their own rules for punctuation signs (English should always use the single unicode character)

**Comment by SoftVision-PaulOiegas** [comment]
Hi @meandavejustice, brought the branch with the fix locally and tried to test the fix, but something is quite strange. There is no "Featured Experiment" section on that branch.
Am I doing something wrong?

**Comment by meandavejustice** [comment]
@SoftVision-PaulOiegas You can just manually set `is_featured` to true in notes.yaml for testing

**Comment by SoftVision-PaulOiegas** [comment]
Uuu, nice! Didn't knew it's that easy to make one of the experiments featured. I've tried in the past but I've searched in totally other files than <experiment>.yaml ones.

The fix seems to be ok. Even tried with browser window resize to mobile view and other negative testing but all went well. 👍

**Pull request merged**